packages/transport-native-ble/src/logs.ts

```diff
@@ -0,0 +1,6 @@
+export const logs: string[] = [];
+
+// This is for debug purposes it should be removed once BT goes to production
+export const log = (message: string) => {
+    logs.push(message);
+};
```

Comment:
one attempt to unify logs is this https://github.com/trezor/trezor-suite/blob/develop/packages/utils/src/logs.ts#L12